include/zephyr/bluetooth/conn.h

```diff
@@ -252,6 +252,93 @@ struct bt_conn_le_subrate_changed {
 	uint16_t supervision_timeout;
 };
 
+/** Most Connection Interval Groups possible
+ *
+ * The practical maximum is 41 groups based on HCI event size constraints:
+ *     (HCI event max size - cmd_complete overhead - response fields) / sizeof(group)
+ *     (255 - 4 - 3) / 6 = 41 groups max
+ */
+#define BT_CONN_LE_MAX_CONN_INTERVAL_GROUPS 41
+
+/** Minimum supported connection interval group
+ *
+ * Each group represents an arithmetic sequence of supported connection intervals:
+ *   min, min+stride, min+2*stride, ..., min+n*stride (where min+n*stride <= max)
+ *
+ * Example: min=10 (1250 us), max=60 (7500 us), stride=5 (625 us)
+ *   -> Supported: 1250 us, 1875 us, 2500 us, ..., 6875 us, 7500 us
+ *
+ * All values in units of 125 microseconds.
+ */
+struct bt_conn_le_min_conn_interval_group {
+	/** Lower bound of group interval range */
+	uint16_t min;
+	/** Upper bound of group interval range */
+	uint16_t max;
+	/** Increment between consecutive supported intervals */
+	uint16_t stride;
+};
+
+/** Minimum supported connection interval information */
+struct bt_conn_le_min_conn_interval_info {
+	/** Minimum supported connection interval in units of 125 microseconds. */
+	uint8_t min_supported_conn_interval;
+	/** Number of interval groups. */
+	uint8_t num_groups;
+	/**
+	 * Array of supported connection interval groups.
+	 *
+	 * Multiple groups allow representing non-contiguous or differently-strided ranges.
+	 * If num_groups=0, only intervals in the Baseline ConnInterval Values (BCV) range
+	 * are supported.
+	 */
+	struct bt_conn_le_min_conn_interval_group groups[BT_CONN_LE_MAX_CONN_INTERVAL_GROUPS];
+};
```

Comment:
This struct is 248 bytes (+1 byte alignment). Will this increase our stack size requirement?